autotest/ogr/ogr_flatgeobuf.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+###############################################################################
+# $Id$
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  FlatGeobuf driver test suite.
+# Author:   Björn Harrtell <bjorn@wololo.org>
+#
+###############################################################################
+# Copyright (c) 2018-2019, Björn Harrtell <bjorn@wololo.org>
+#
+# This library is free software; you can redistribute it and/or
```

Comment:
Please use the X/MIT header for that. Use of LGPL in some of the .py script is an historic accident